src/main/kotlin/com/ampnet/mailservice/service/impl/mail/AbstractMail.kt

```diff
@@ -73,13 +77,23 @@ abstract class AbstractMail(
             }
         }
 
-    private fun fillTemplate(): String {
+    private fun fillTemplate(template: Mustache): String {
         val writer = StringWriter()
         template.execute(writer, data).flush()
         return writer.toString()
     }
+
+    private fun getLanguageData(language: String): LanguageData =
+        languageData.firstOrNull { it.language == language } ?: languageData.first { it.language == DEFAULT_LANGUAGE }
+
+    data class LanguageData(
+        val language: String,
```

Comment:
Maybe extract languages to separate enum, "en" hardcoded is repeated in several classes.